tools/tdx-measure/rtmr/rtmr.go

```diff
@@ -215,6 +216,13 @@ func CalcRtmr0(firmware []byte) ([48]byte, error) {
 	// - 0x0000: disabled
 	// - 0x0001: enabled
 	rtmr.extendVariableValue([]byte{0x00, 0x00})
+	if withGPU {
+		// QEMU FW CFG.opt/ovmf/X-PciMmio64Mb (GPU only, set by Kata to a static value)
+		// See:
+		// - https://github.com/kata-containers/kata-containers/blob/c7d0c270ee7dfaa6d978e6e07b99dabdaf2b9fda/src/runtime/virtcontainers/qemu.go#L817-L827
+		// - https://github.com/kata-containers/kata-containers/blob/c7d0c270ee7dfaa6d978e6e07b99dabdaf2b9fda/src/runtime/virtcontainers/qemu_arch_base.go#L877-L908
+		rtmr.extendVariableValue([]byte("524288"))
```

Comment:
This is the memory size in MiB, right? We could make that an input, instead of the bool. On the command line, we could pass the model and look up its memory.

Reply:
It's a little more contrived, I think. The BAR size is a somewhat scaled-up version of the actual GPU memory size. Will investigate tomorrow.